### DISCUSSION TITLE
Regex for Mac Shares to ignore/clean ._

### DISCUSSION THREAD

**Discussion by phrohike** [stashapp/stash#2829] [Q&A] [answered]
Hey All, 

Mac makes dupes in SMB shares (and on the local drive) using ._ before the file name to store extra info about files. Does anyone know a regex value I could add to the Exclusions list to ignore these files (or clean up the files already imported)? I've tried multiple regex combinations but can't seem to figure the right one out. Thanks for any help.

**Comment by phrohike** [comment]
I forgot to add that the file names have the ._ added to the beginning of the file name like this:   ._08bcac66-a39e-42f4-9ae8-b31886bb9490.jpeg, ._kini_05.jpeg, ._medium.jpeg, etc. Thanks.

**Comment by MrX292** [comment, marked as answer]
`.+/\._.+\.jpeg$` should work for jpeg
and `.+/\._.+\..+$` for all file extensions
you can test them in stash with filter  path - matches regex

**Reply by phrohike** [reply]
That worked! Thank you!
